splash/tests/mockserver.py

```diff
@@ -764,7 +764,7 @@ class Image(Resource):
         @use_chunked_encoding
         def render_GET(self, request):
             request.setHeader(b"Content-Type", b"image/gif")
-            return base64.decodestring(b'R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=')
+            return base64.decodebytes(b'R0lGODlhAQABAAD/ACwAAAAAAQABAAACADs=')
```

Comment:
We're still trying to support Python 2.x; it looks like in 2.x there is not base64.decodebytes method.